ui/imports/shared/controls/chat/ProfileHeader.qml

```diff
@@ -155,6 +155,7 @@ Item {
         Item {
             Layout.fillWidth: true
             implicitHeight: displayNameLabel.implicitHeight
+            visible: root.displayNameVisible
```

Comment:
@igor-sirotin had to fix a little regression here :) the displayName was always visible